digital_milliet/lib/author_builder.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, db=None, catalog=None, collection_name="annotation"):
         self.mongo = db
         self.catalog = catalog
         self.__collection_name__ = collection_name
+        self.app = app
+        if app and "AUTHORS_COLLECTION" in self.app.config:
```

Comment:
hmm. it would be nice to make the collection name configurable for the commentaries and mirador annotations too